package.json

```diff
@@ -136,6 +136,7 @@
       "no-console": 1,
       "no-use-before-define": 0,
       "prefer-template": 2,
+      "class-methods-use-this": 0,
```

Comment:
I don't feel the need to use static on class methods that aren't using `this`.

http://eslint.org/docs/rules/class-methods-use-this